fr.henri2h.minestrix.desktop

```diff
@@ -0,0 +1,8 @@
+[Desktop Entry]
+Type=Application
+Name=MinesTRIX
+Icon=fr.henri2h.minestrix
+Exec=env GDK_GL=gles minestrix %U
```

Comment:
this is likely incorrect.

Reply:
I removed the env GDK_GL=gles which didn't made any sense indeed